docs/eventing/samples/helloworld/helloworld-python/README.md

```diff
@@ -0,0 +1,338 @@
+A simple web app written in Python that you can use to test knative eventing. It shows how to consume a [CloudEvent](https://cloudevents.io/) in Knative eventing, and optionally how to respond back with another CloudEvent in the http response, by adding the Cloud Eventing headers outlined in the Cloud Events standard definition.
+
+We will deploy the app as a [Kubernetes Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) along with a [Kubernetes Service](https://kubernetes.io/docs/concepts/services-networking/service/).
+However, you can also deploy the app as a [Knative Serving Service](../../../../serving/README.md).
+
+Follow the steps below to create the sample code and then deploy the app to your
+cluster. You can also download a working copy of the sample, by running the
+following commands:
+
+```shell
+# Clone the relevant branch version such as "release-0.13"
+git clone -b "{{< branch >}}" https://github.com/knative/docs knative-docs
+cd knative-docs/docs/eventing/samples/helloworld/helloworld-python
+```
+
+## Before you begin
+
+- A Kubernetes cluster with [Knative Eventing](../../../getting-started.md#installing-knative-eventing) installed.
+- [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured (we'll use it for a container registry).
+
+## Recreating the sample code
+
+1. Create a new file named `helloworld.py` and paste the following code. This
+   code creates a basic web server which listens on port 8080:
+
+    ```python
+    from flask import Flask, request, make_response
+    import uuid
+
+    app = Flask(__name__)
+
+    @app.route('/', methods=['POST'])
+    def hello_world():
+        app.logger.warning(request.data)
+        # Respond with another event (optional)
+        response = make_response({
+            "msg": "Hi from helloworld-python app!"
+        })
+        response.headers["Ce-Id"] = str(uuid.uuid4())
+        response.headers["Ce-specversion"] = "0.3"
+        response.headers["Ce-Source"] = "knative/eventing/samples/hello-world"
+        response.headers["Ce-Type"] = "dev.knative.samples.hifromknative"
+        return response
+
+    if __name__ == '__main__':
+        app.run(debug=True, host='0.0.0.0', port=8080)
+
+
+    ```
+
+1. Add a requirements.txt file containing the following contents:
+
+    ```bash
+    Flask==1.1.1
+    ```
+1. In your project directory, create a file named `Dockerfile` and copy the code
+   block below into it. For detailed instructions on dockerizing a Go app, see
```

Comment:
Maybe we should replace the "dockerizing a go app" section with something related to Python, or possibly just remove the line related to Go?